src/locale/pages/assignees/en.yml

```diff
@@ -0,0 +1,7 @@
+taskStates:
+    editable: When published, this task will be assigned to the people matching the targeting Smart Search query
+    publishing: This task is currently being assigned to all that match the targeting Smart Search query...
+    published: This task has been assigned to all people that match the targeting Smart Search query.
+links:
+    readOnly: View the query in read-only mode.
```

Comment:
I think _"View the query"_ (or "View the Smart Search") is enough, without explicitly pointing out here that it's read-only.